README.md

```diff
@@ -18,6 +20,14 @@ You can try out the user-side of Calypso on [WordPress.com](https://wordpress.co
 4.	Execute `make run` or `make dashboard` (for a more visually-oriented interface) from the root directory of the repository.
 5.	Open `calypso.localhost:3000` in your browser.
 
+### Docker
+
+1. Download [Docker for Mac](https://docs.docker.com/docker-for-mac/), [Docker for Windows](https://docs.docker.com/docker-for-windows/) unless you are in linux, in which case, just install [Docker](https://docs.docker.com/engine/installation/linux/)
+2. Add `127.0.0.1 calypso.localhost` to your local hosts file.
+2. Run `docker-compose run --rm -p 3000:3000 calypso`
```

Comment:
we should not need `-p` here since it's specified in the `docker-compose.yml` file